public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -981,7 +981,7 @@ export class LokiDatasource
   // Used when running queries through backend
   applyTemplateVariables(target: LokiQuery, scopedVars: ScopedVars): LokiQuery {
```

Comment:
If `scopedVars` can be undefnied, shouldn't we update the type here?
Additionally, it would have been ideal to have a regression test for this bug.